### DISCUSSION TITLE
Prisma findMany in every week

### DISCUSSION THREAD

**Discussion by mtbagus** [prisma/prisma#18566] [Q&A]
this is was the query :
http://localhost:3000/users/logs/week?week=13&year=2023
is it possible to get a data at a same week in this query?
```
const value = req.query
const result = await prisma.test.findMany({
                where: {
                    isActive: true,
                    week: value.week,
                    year: value.year
                },
            });
            render(res, 200, statuscodes.OK, result);
```
because when i try to hit this endpoint it always return an empty array.
idk how to solve this maybe someone know how to use it?

**Comment by ludralph** [comment]
Hi @mtbagus 👋 

Can you please share the relevant schema for this query to help us understand this better?

**Comment by mtbagus** [comment]
im so sorry, its been working rn thanks
